assets/tests/components/detours/detourMap.test.tsx

```diff
@@ -224,4 +229,44 @@ describe("DetourMap", () => {
       container.querySelector("img[src^=test_satellite_url")
     ).not.toBeNull()
   })
+
+  test("stops are drawn on the map", () => {
+    const stops = [
+      { ...stopFactory.build(), missed: false },
+      { ...stopFactory.build(), missed: false },
+      { ...stopFactory.build(), missed: false },
+    ]
+
+    const { container } = render(<DetourMapWithDefaults stops={stops} />)
+
+    expect(stopIcon.getAll(container)).toHaveLength(3)
+  })
+
+  test("stops are drawn as missed stop icons when they are missed", () => {
+    const stops = [
+      { ...stopFactory.build(), missed: true },
+      { ...stopFactory.build(), missed: true },
+      { ...stopFactory.build(), missed: false },
+    ]
+
+    const { container } = render(<DetourMapWithDefaults stops={stops} />)
+
+    expect(stopIcon.getAll(container)).toHaveLength(1)
+    expect(missedStopIcon.getAll(container)).toHaveLength(2)
+  })
+
+  test("duplicate stops are only drawn once", () => {
+    const duplicateStop = { ...stopFactory.build(), missed: true }
+    const stops = [
+      { ...stopFactory.build(), missed: true },
+      duplicateStop,
+      { ...stopFactory.build(), missed: false },
+      duplicateStop,
+    ]
```

Comment:
suggestion(non-blocking): [I thought I posted this already but I can't find it, so sorry if this is a duplicate] I'm not sure if it's important that this test data match real world data, but since we can only have contiguous missed stops right now, maybe this test data should match?


```suggestion
    const duplicateStop = { ...stopFactory.build(), missed: true }
    const stops = [
      duplicateStop,
      { ...stopFactory.build(), missed: true },
      duplicateStop,
      { ...stopFactory.build(), missed: false },
    ]
```

OR


```suggestion
    const duplicateStop = stopFactory.build()
    const stops = [
      { ...stopFactory.build(), missed: true },
      { ...duplicateStop, missed: true },
      { ...stopFactory.build(), missed: false },
      { ...duplicateStop, missed: false },
    ]
```